.ruby-version

```diff
@@ -1 +1 @@
-2.7.0
+3.1.2
```

Comment:
Are we sure this is safe? I thought we pinned to ruby 2.7 was to keep in step with the AWS lambda ruby version

Reply:
Ah right, my mistake, I hadn't realised that was the hard limit. I'll revert this change.